src/components/buttons/SliderButton.tsx

```diff
@@ -1,25 +1,23 @@
-import Image from 'next/image';
-
 import classNames from 'classnames/bind';
 
-import { SVGS } from '@/constants';
+import { SVGS } from '@/constants/images';
```

Comment:
```suggestion
import { SVGS } from '@/constants/index';
```
구두로도 말씀드렸지만 이 constants, types 전부 index에서 import하도록 수정하시면 좋을 것 같습니다~